### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by patel-prakashkumar**
Hi,
I have finished JavaScript1 week1 Assignment, I have added two file and Created Pull request.
Thanks

### CODE REVIEW

File: javascript/javascript1/week1/myscripts.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+// Task 1
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js]: Check Template literals and how you can write these statements in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: javascript/javascript1/week1/myscripts.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+// Task 2
+let dogYearOfBirth = 2017;
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js]: Why did you choose to use let for all four variables?

File: javascript/javascript1/week1/myscripts.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+let dogYearFuture = 2027;
+let dogYear = dogYearFuture - dogYearOfBirth + 60;
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js]: why 60? I think `const dogYear = (dogYearFuture - dogYearOfBirth) * 7`

File: javascript/javascript1/week1/myscripts.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+let shouldShowResultInDogYear = true;
+
+if (shouldShowResultInDogYear == true) {
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js]: `if (shouldShowResultInDogYear)` is enough

File: javascript/javascript1/week1/myscripts.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  console.log(
+    "Your dog will be " + dogYear + " dog years old in " + dogYearFuture
+  );
+} else {
+  let humanYear = dogYear - 60;
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js]: `const dogYearInHumanYear = dogYear / 7;`

Comment (cecastosic) [reply javascript/javascript1/week1/myscripts.js]: and you can also put it next to other defined variables, after `dogYear` - line 11, and before `shouldShowResultInDogYear`

File: javascript/javascript1/week1/myscripts.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+// Task 3
+// Peter house prise calculation
+let peter = { wide: 8, deep: 10, high: 10, gardenSizeInM2: 100 };
+let peterVolumeInMeters = peter.wide * peter.deep * peter.high;
+let peterHousePrise =
+  peterVolumeInMeters * 2.5 * 1000 + peter.gardenSizeInM2 * 300;
+if (peterHousePrise > 2500000) {
+  console.log("Peter is paying too much");
+} else {
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js:30]: what if `peterHousePrice === 2500000`?

File: javascript/javascript1/week1/myscripts.js [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+// Julia house prise calculation 
+let julia = { wide: 5, deep: 11, high: 8, gardenSizeInM2: 70 };
+let juliaVolumeInMeters = julia.wide * julia.deep * julia.high;
+let juliaHousePrise =
+  juliaVolumeInMeters * 2.5 * 1000 + julia.gardenSizeInM2 * 300;
+
+if (juliaHousePrise > 1000000) {
+  console.log("Julia is paying too much");
+} else {
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js:42]: what if `juliaHousePrice === 1000000 `?

File: javascript/javascript1/week1/myscripts.js [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+  console.log(
+    "Your dog will be " + humanYear + " human years old in " + dogYearFuture
+  );
+};
+
+// Task 3
+// Peter house prise calculation
+let peter = { wide: 8, deep: 10, high: 10, gardenSizeInM2: 100 };
+let peterVolumeInMeters = peter.wide * peter.deep * peter.high;
+let peterHousePrise =
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js]: all `const` ;)

File: javascript/javascript1/week1/myscripts.js [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+if (peterHousePrise > 2500000) {
+  console.log("Peter is paying too much");
+} else {
+  console.log("Peter is paying too little");
+};
+
+// Julia house prise calculation 
+let julia = { wide: 5, deep: 11, high: 8, gardenSizeInM2: 70 };
+let juliaVolumeInMeters = julia.wide * julia.deep * julia.high;
+let juliaHousePrise =
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js]: all `const` ;)

File: javascript/javascript1/week1/myscripts.js [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+  "Handly",
+  "Data",
+  "Vital",
+  "Alpha",
+  "Cross",
+  "Serva",
+];
+const randomNumber = Math.floor(Math.random() * 10);
+const startupName = firstWords[randomNumber] + secondWords[randomNumber];
+console.log(
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js:75]: Please use Prettier 🙂

File: javascript/javascript1/week1/myscripts.js [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+  "FlowFreer",
+  "RypeMake",
+  "Handly",
+  "Data",
+  "Vital",
+  "Alpha",
+  "Cross",
+  "Serva",
+];
+const randomNumber = Math.floor(Math.random() * 10);
```
Comment (cecastosic) [inline javascript/javascript1/week1/myscripts.js]: What if you need two random numbers per array? 🙂

**Review by cecastosic — COMMENTED** [review]
Great job @patelprakash0508 💪 I like the way you solved the house pricing task. Please be careful when to use `let` and when to use `const`. Maybe this video can help https://youtu.be/1mgLWu69ijU Try to use English names for variables. (price instead of prise for example).  🌸

File: javascript/javascript1/week1/myscripts.js (hunk 1 above)
Comment (patel-prakashkumar) [reply javascript/javascript1/week1/myscripts.js]: okay.. I have read document, I will use single quote in string.

File: javascript/javascript1/week1/myscripts.js (hunk 3 above)
Comment (patel-prakashkumar) [reply javascript/javascript1/week1/myscripts.js]: Oky, its miss understood concept, I have just see in assignment difference between human year and dog year. now I have cleared, I can do it by multiply 7.

File: javascript/javascript1/week1/myscripts.js (hunk 6 above)
Comment (patel-prakashkumar) [reply javascript/javascript1/week1/myscripts.js:30]: Its will come wrong result. for that condition, I have to use peterHousePrise >=

File: javascript/javascript1/week1/myscripts.js (hunk 11 above)
Comment (patel-prakashkumar) [reply javascript/javascript1/week1/myscripts.js]: I have create solution below, could you please check, Is it right solution or wrong?
const startupName = [firstWords[randomNumber1] + secondWords[randomNumber1]] + " " +  [firstWords[randomNumber2] + secondWords[randomNumber2]];

File: javascript/javascript1/week1/myscripts.js (hunk 1 above)
Comment (cecastosic) [reply javascript/javascript1/week1/myscripts.js]: It is not a single quote but backtick

File: javascript/javascript1/week1/myscripts.js (hunk 11 above)
Comment (cecastosic) [reply javascript/javascript1/week1/myscripts.js]: ```const startupName = `${firstWords[randomNumber1]} ${secondWords[randomNumber2]}``` should be enough

File: javascript/javascript1/week1/myscripts.js (hunk 1 above)
Comment (patel-prakashkumar) [reply javascript/javascript1/week1/myscripts.js]: oky. could please also guide me, How can I replace all double quote to backtick in vs code editor.

File: javascript/javascript1/week1/myscripts.js (hunk 11 above)
Comment (patel-prakashkumar) [reply javascript/javascript1/week1/myscripts.js]: I have tried but its come error and last task and rest of the code I have changed in JS file according to comments. I have also cleared backtick concept so I don't have to use many time in double quote in line and also we can assign variable by $ sign with {}  in between two string.

**Comment by patel-prakashkumar** [comment]
I have updated task 4. I have made small mistake that's why come error.

File: javascript/javascript1/week1/myscripts.js (hunk 1 above)
Comment (cecastosic) [reply javascript/javascript1/week1/myscripts.js]: I am not sure I understand your question. In this particular case you should write 
```console.log(`You will be ${age} years old in ${yearFuture}`);```
What is backtick? - good article https://www.computerhope.com/jargon/b/backquot.htm#:~:text=Alternatively%20known%20as%20acute%2C%20backtick,keyboard%20key%20as%20the%20tilde.

**Review by cecastosic — APPROVED** [review]

**Pull request merged**